lib/Cake/Test/Case/View/Helper/HtmlHelperTest.php

```diff
@@ -553,6 +553,31 @@ public function testImageTagWithTheme() {
 		}
 	}
 
+	/**
+	 * testBase64ImageTag method
+	 *
+	 * @return void
+	 */
+	public function testBase64ImageTag()
+	{
+		$this->Html->request->webroot = '';
+		
+		$result = $this->Html->image('cake.icon.png', ['base64' => true]);
+		$this->assertTags($result, array(
+			'img' => array(
+				'src' => 'preg:/data:image\/png;base64,(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})/',
+				'alt' => ''
+			)));
+
+		$result = $this->Html->image('/img/cake.icon.png', ['base64' => true]);
+		$this->assertTags($result, array(
+			'img' => array(
+				'src' => 'preg:/data:image\/png;base64,(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})/',
+				'alt' => ''
+			)));
+
+	}
```

Comment:
What about the case where the file doesn't exist? As per your initial comment, I think an exception _is_ the right choice here.